avro2py/codegen.py

```diff
@@ -408,6 +408,25 @@ def visit_AnnAssign(self, node: ast.AnnAssign) -> ast.AnnAssign:
             return rewriter.visit(node)
 
 
+def resolve_schema_reference(schema: Record, schemas : List[Record]):
+    """Substitute the schema reference with the definition.""" 
+    if not schemas:
+        return
+
+    for field in schema.resolved_schema['fields']:
+        if isinstance(field['type'], str) and field['type'].find('.') >= 0:
```

Comment:
When is `field['type']` not going to be a string? Also, isn't the latter part of this just `'.' in field['type']`?

Reply:
In case of nested schema `field['type']` will be a dict.
Yes, I will change the latter part with your suggestion. It looks much more readable.